15 tests passed.
3 tests failed.
#1.  One of the test cases timed out, probably due to an inefficient solution (testLarge1Heap(PrivateTests))
#2.  One of the test cases timed out, probably due to an inefficient solution (testLarge2Heap(PrivateTests))
#3.  One of the test cases timed out, probably due to an inefficient solution (testLarge3Heap(PrivateTests))
